### PR TITLE
fix: Simplified aggregation and optimised version : Updated aggregation.py

### PR DESCRIPTION
**Pull request by RILUCK**
**What this PR does / why we need it**:
It simplified the complex unnecessary logic in aggregate.py file and optimised the code faster.

**Which issue(s) this PR fixes**:

Fixes #
Remove unnecessary and slow logic.

### CODE REVIEW

**Comment by RILUCK** [comment]
This file has been updated for fast processing
File PATH:
sdk/python/feast/aggregation.py

**Comment by RILUCK** [comment]
@adchia Please review the optimised code and approve after test.

**Comment by RILUCK** [comment]
DCO is failing even after fixing the issue using instructions. Please check/test and approve :)

File: test [hunk 1]
```diff
@@ -0,0 +1 @@
+optimised aggregate
```
Comment (adchia) [inline test:1]: can you revert this?

File: setup.py [hunk 2]
```diff
@@ -390,4 +390,4 @@ def run(self):
         "build_py": BuildCommand,
         "develop": DevelopCommand,
     },
-)
+)
```
Comment (adchia) [inline setup.py:393]: add back the new line?

**Review by adchia — COMMENTED** [review]
/ok-to-test

File: feast
Comment (adchia) [inline feast:1]: can you revert this submodule?

File: sdk/python/feast/aggregation.py [hunk 3]
```diff
@@ -81,13 +81,12 @@ def from_proto(cls, agg_proto: AggregationProto):
...
-        return True
+        
+        # Optimised return logic here
+        return (
+            self.column == other.column and 
+            self.function == other.function and 
+            self.time_window == other.time_window and 
+            self.slide_interval == other.slide_interval
+        )
+
```
Comment (adchia) [inline sdk/python/feast/aggregation.py:92]: can you fix this "no newline" bug

**Pull request closed without merging**